src/views/dashboard/styles.css

```diff
@@ -2,11 +2,71 @@
   display: flex;
   flex-wrap: wrap;
   flex-direction: row;
-  align-items: stretch;
+  margin: 15px;
 }
 
 .item {
   flex-basis: calc(25% - 20px);
   align-self: center;
   margin: 10px;
+  border: 1px solid;
+  border-radius: 5px;
+  min-height: 128px;
+}
+
+.userCreated {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: rgba(210, 210, 210, 0.75);
+  height: 50px;
+}
+
+.crawler {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: rgba(243, 155, 56, 0.92);
+  height: 40px;
+}
+
+@keyframes crawlerFinished {
+  0% {
+    background: rgba(243, 155, 56, 0.92);
+    height: 40px;
+  }
+  25% {
+    background: rgba(16, 193, 16, 0.92);
+    font-size: inherit;
+    height: 40px;
+  }
+  80% {
+    background: rgba(16, 193, 16, 0.92);
+    font-size: inherit;
+    opacity: 1;
+    height: 40px;
+  }
+  100% {
+    font-size: 0;
+    height: 0;
+    opacity: 0;
+    display: none;
+  }
+}
+
+.crawlerRunning::after {
+  content: "UPDATING DASHBOARD - please wait";
```

Comment:
I don't think would become a problem in this project but generally speaking this might lead to `l18n` issues.. Ideally you'd do this with html (jsx), not css.